organization/roadmap.md

```diff
@@ -0,0 +1,70 @@
+---
+type: Resource
+---
+
+# Roadmap
+
+This roadmap serves as a model for how we're developing, growing and scaling our organization. It sets out the roles for both the staff and the members of the Foundation for Public Code.
+
+## Phase 0: Get the organization founded
+
+Staff of the Foundation for Public Code will:
+
+* [Build team](../roles/)
```

Comment:
should this link not be (../roles/index.md) ?

Reply:
Yes, but that's too big a change for this review. I've logged it.